topicgenerator/src/main/java/io/dockstore/topicgenerator/helper/StringFilter.java

```diff
@@ -0,0 +1,33 @@
+package io.dockstore.topicgenerator.helper;
+
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+
+public interface StringFilter {
+
+    /**
+     * @param topic
+     * @return true on failure  (i.e. should be filtered out and is caught by the filter)
+     */
+    boolean assessTopic(String topic);
```

Comment:
Conceptually, this method works the opposite way that the java stream methods do (they keep the elements where the predicate evaluates to true).  Is there a name that better conveys that the method returns true if the string is "bad"?